gabsync/Dockerfile

```diff
@@ -0,0 +1,72 @@
+ARG docker_repo=zokradonh
+FROM ${docker_repo}/kopano_base
```

Comment:
I would love to name it kopano_gabsync because of the container = micro service idea. If we need some other sync tool later on it should be deployed in a separate container aswell.